app/scss/_issue.scss

```diff
@@ -55,31 +55,27 @@
   img {
     width: $thumbnail-width;
     height: auto;
-    margin-bottom: $spacing-base;
+    margin: 0 $spacing-base $spacing-base 0;
     border: 1px solid $color-light-gray;
     box-shadow: $box-shadow2;
 
     @include bp(screen1) {
       margin: 0 $spacing-md 0 0;
-      grid-row: 1 / 4;
+      grid-row: 1 / span 3;
```

Comment:
I'd like a walkthrough of this please. It doesn't have to delay the PR, but I'm real curious what this even means.